system/CLI/CLI.php

```diff
@@ -147,7 +147,7 @@ class CLI
 	 *
 	 * @var string
 	 */
-	protected static $lastWrite;
+	protected static $lastWrite = 'write';
```

Comment:
What were you running into here? Lastwrite should be null before it's used so that an initial call to 'write' will add an empty line.

Reply:
The problem is that the previous unit tests were not looking for that empty line, and hence failed on mmy system. Basically, the previous PR broke that unit test on my system.